src/Illuminate/Console/ConfirmableTrait.php

```diff
@@ -48,7 +48,7 @@ public function confirmToProceed($warning = 'Application In Production!', $callb
     protected function getDefaultConfirmCallback()
     {
         return function () {
-            return $this->getLaravel()->environment() === 'production';
+            return $this->getLaravel()->isProduction();
```

Comment:
@mvmx @driesvints This change breaks the application interface and should be reverted.

Reply:
Reverted.